scripts/modelbench/launch.sh

```diff
@@ -85,7 +85,9 @@ if [ $TORCH_START_COMMIT == $TORCH_END_COMMIT ]; then
     docker exec -i $USER bash -c "bash version_collect.sh $LOG_DIR $DYNAMO_BENCH"
 
     if [ $TEST_MODE == "inference" ]; then
-        docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE $LOG_DIR $WRAPPER $HF_TOKEN $BACKEND $EXTRA"
+        docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE $LOG_DIR $WRAPPER $HF_TOKEN $BACKEND inference $SUITE $EXTRA"
```

Comment:
also need add `suite` in those 2 lines

Reply:
already added at the end of this line.`inference $SUITE $EXTRA"`